### DISCUSSION TITLE
Welcome to fmod-gdnative Discussions!

### DISCUSSION THREAD

**Discussion by piiertho** [utopia-rise/fmod-gdextension#126] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bitbrain** [comment]
My name is Miguel and I have been dabbling with software engineering since 2008. I started off with Delphi Pascal, then quickly moved to C++ and then learned Java. During university I'd attempt to build games with libgdx (a Java game library) but after a couple of years I noticed that I did spend more time on writing tools than doing actual game development. Therefore, I picked Godot Engine as my new home and have been building games with it ever since. My latest project is an RPG featuring dwarves, goblins and other things (and I've been documenting the development of it on my Youtube channel).

I am collaborating with two friends of mine who happen to be audio engineers and I quickly realised that we need some solution to quickly iterate on sound design and music. Initially, they would upload .ogg files and push them to the Github repository. I'd then go and manually implement the sound effects and music. Sometimes they wanted a slightly different change or add additional sound effects during gameplay. Every single time I had to go into the code and adjust things. It was very frustrating for everyone: I had to do this manual migration, while the two guys were unable to really preview and iterate on their work.

[Then I discovered FMOD](https://www.twitch.tv/videos/248998904) and I was blown away how PowerupAudio was able to facilitate the tool to edit music in real-time while the game was running. I stumbled upon this addon for Godot and got it working rather quickly. Nowadays, FMOD is integrated into my RPG and the sound peeps can simply edit the audio in real-time while the game is running. I'll never go back to anything else!
